cmake/std/atdm/mutrino/environment.sh

```diff
@@ -51,7 +51,7 @@ if [ "$ATDM_CONFIG_COMPILER" == "INTEL" ] && [ "$ATDM_CONFIG_KOKKOS_ARCH" == "HS
 elif [ "$ATDM_CONFIG_COMPILER" == "INTEL" ] && [ "$ATDM_CONFIG_KOKKOS_ARCH" == "KNL"  ]; then
     module use /projects/EMPIRE/mutrino/tpls/knl/modulefiles
     export SLURM_TASKS_PER_NODE=16
-    export OMP_NUM_THREADS=16
```

Comment:
Wow, I can't believed I messed that when I was looking at this.  Going for 16 down to 2 threads per process should make a huge difference.